### ISSUE TITLE
Using sentinel config fails with journal queries

### ISSUE THREAD

**Issue by jonfox** [safety-data/akka-persistence-redis#16]
When I configure the plugin to use sentinel, I get failures on journal queries. The problem seems to be that in `EventsByPersistenceId`, `EventsByTagSource`, and `PersistenceIdsSource`, the code in `preStart()` to create the subscription assumes the standard config, rather than sentinel. I have it working by changing:

```
        subscription = RedisPubSub(host = RedisUtils.host(conf),
          port = RedisUtils.port(conf),
          channels = Seq(identifiersChannel),
          patterns = Nil,
          authPassword = RedisUtils.password(conf),
          onMessage = callback.invoke)(system)
```

to

```
        subscription = RedisPubSub(host = redis.host,
          port = redis.port,
          channels = Seq(identifiersChannel),
          patterns = Nil,
          authPassword = redis.password,
          onMessage = callback.invoke)(system)
```

Does that fix seem reasonable?

Thanks -

**Comment by satabin** [comment]
Hi,

Indeed, I wrote the journal query part as if only standard configuration exists.
Reusing the redis settings seems the best way to do it instead of assuming it is standard config.

You can either open a pull request or I will do it. as you wish.

**Comment by jonfox** [comment]
Thanks, see #17

**Comment by satabin** [comment]
Closed by #17

**Comment by satabin** [comment]
0.3.0-SNAPSHOT updated with this fix. 0.3.0 release planned on Thursday.
